src/client/components/FavoriteList/FavoriteList.component.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './FavoriteList.styles.css';
+import ProductImage from './ProductImage.js';
+import ProductQuantity from './ProductQuantity';
+import ProductCart from './ProductCart';
+
+export default function FavoriteList( props ) {
+  
+  const {favorites, setFavorites} = props;
```

Comment:
Agree with @Divyajg. A common naming convention when passing event handler functions like this in React is to call it like this:

```suggestion
  const {favorites, handleSetFavorite} = props;
```

You're basically passing this prop around all over your components so it does not make it any less confusing with this name 🙈